blocks/core/ff_module/ff_module-filter/ff_module-filter.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+$(function(){ 
+	$("[data-control='filter-dropdown']").click(function(){
+		var filterDropdown = $(this),
+		      filterIcon = filterDropdown.find("[data-icon='filter-dropdown-icon']"),
+		      filterContent = filterDropdown.next();
```

Comment:
I don't think it's a good idea to use .next() in the module js, as it ties the code to the structure of the html, which may change. Instead, the initiating data-* declaration should be at the parent element of all effected elements, and each essential element should have the data-* attribute that indicates its function and also its relation to the overall module. @roobottom perhaps we should come up with a js naming convention too? Or invent jsBEM? :)